libraries/ESP8266WebServer/examples/HelloServerSecure/HelloServerSecure.ino

```diff
@@ -0,0 +1,136 @@
+#include <ESP8266WiFi.h>
```

Comment:
Likewise, need a license header, a brief description what this sketch does, and instructions for generating certificate and key arrays.

Also, a note that browsers will not (by default) trust a self-signed certificate might be useful.

Reply:
Done (see above).